### PR TITLE
Removed alt text from Homepage

### PR DESCRIPTION
**Pull request by RAidanM**
Most alt text is now alt=""
Improved alt text on the site logo.

### CODE REVIEW

File: scripts/json-lint.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def assert_list_image(img: dict):
     assert(isinstance(img['imageSrc'], str))
     image_exists(img['imageSrc'])
     assert(isinstance(img['imageAlt'], str))
-    assert(img['imageAlt'])
+    #assert(img['imageAlt'])
```
Comment (JonEsparaz) [inline scripts/json-lint.py:36]: This assertion is still important and should be modified to allow for empty strings:

```suggestion
    assert(isinstance(image['imageAlt'], str))
```

Also, if you are removing code in the future please delete it (don't just comment it out).

**Review by JonEsparaz — CHANGES_REQUESTED** [review]

**Comment by JonEsparaz** [comment]
(Please ignore me if these changes were discussed and agreed upon).

Null alt text means that the images are purely decorative and do not provide any information, but it is my understanding that the images chosen for the website (especially the homepage) help tell a story and show off the TMH vibe/culture. If that's the case, I think well-written alt text is necessary to keep the experience uniform for all visitors.

@deeinstereo, do you have an opinion on this?

**Comment by GeorgeBellTMH** [comment]
I kinda agree with @JonEsparaz ... I'm wondering if we have anyone we could reach out to who could give us some advice on this...it seems like this would be useful, but I'd like to know what people would normally expect... @lucastbelem do we have anyone who could give us some advice when it comes to stuff like this?

**Comment by JonEsparaz** [comment]
Feel free to consult someone with more expertise on accessibility standards, but here's my final input: web accessibility is about providing the same experience for all users. Ask yourself this question: would the website be the same to you without the images?

You'd be hard-pressed to find any well-built websites with null alt text on all images (especially on the homepage).

TL;DR: please keep well-written alt text on the images.

**Comment by lucastbelem** [comment]
@JonEsparaz thanks for the input dude. That's really helpful stuff and I feel the same way - your question about whether the site is the same without the images is a good one - and frankly, it wouldn't be. The images do indeed tell a story. I'm gonna bring one more voice to the table to weigh in before we give it a final thumbs up and include alt text for these images, but I'm convinced we should. 

@mmacedom would you mind reading through this thread and add any thoughts you might have?

Thanks team! Really appreciate working with a team who genuinely cares about accessibility.

**Comment by RAidanM** [comment]
I've been using this guide that Lucas had sent me, ( https://www.w3.org/WAI/WCAG21/quickref/ ), to check the specifications of alt text (sec. 1.1.1 in the doc is about alt text, F67 is the failure code for the issue).

https://webaim.org/techniques/alttext/ is a site that I've seen referenced quite a few places as a guide on common practices when dealing with a site alt text.

It mentions that images that don't contain important content shouldn't have alt text attached to them. A strategy that is suggested is to replace the image with descriptive text only. If that text adds applicable content to the section then that image should have alt text.

I would argue that most images on the website's homepage don't add content to their corresponding text areas.

image

In the example above, using a screen reader, the Go information would be readout then paired with 
"Group of friends sitting outside and having spiritual conversations."

Personally, I found it confusing while navigating the site with my screen off to hear descriptions of pictures that were not providing new information and made it feel more cluttered/redundant.

It's very possible that I am incorrect about this but I just wanted to share what my thought process was while removing pre-existing alt text.

**Comment by mmacedom** [comment]
@lucastbelem these are great questions and great points and it is indeed very important to ensure that the content is accessible to all users. The use of "Alt text" is to purely describe the images and not enhance content. The feature is meant for visually impaired users to hear an accurate description of an image such as "Group of friends sitting outside and having spiritual conversations." 

Perhaps, the actual descriptions can be revised so they are more accurate but they are not meant to enhance the content. 

Hope this helps. 

Here's a good resource to clarify more questions about accessibility https://www.nngroup.com/articles/accessible-design-for-users-with-disabilities/

**Comment by GeorgeBellTMH** [comment]
Closing...
